test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -224,13 +172,127 @@ public void setValueAt(Object aValue, int row, int column) {
     }
 
     public static void main(String[] args) throws Exception {
-        SwingUtilities.invokeAndWait(() -> {
-            try {
-                initTest();
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        });
-        passFailJFrame.awaitAndCheck();
+        String INSTRUCTIONS = """
+                Instructions to Test:
+                1. Refer the below keyboard navigation specs
+                 (referenced from bug report 4112270).
+                2. Check all combinations of navigational keys in all four modes
+                  shift and control verifying each change to the selection against
+                  the spec. If it does, press "pass", otherwise press "fail".
+
+                """;
+
+        INSTRUCTIONS += getOSSpecificInstructions();
+        PassFailJFrame.builder()
+                .instructions(INSTRUCTIONS)
+                .rows(11)
+                .columns(50)
+                .testUI(KeyBoardNavigation::initTest)
+                .build()
+                .awaitAndCheck();
+    }
+
+    public static String getOSSpecificInstructions() {
+        final String WINDOWS_SPECIFIC = """
+                Tab, Shift-Tab - Navigate In.
+                Return/Shift-Return - move focus one cell down/up.
```

Comment:
```suggestion
                Return/Shift-Return - Move focus one cell down/up.
```
For consistency please ensure each command action to start with either lower case or upper case. Same for F2, Esc commands etc.

Check for Linux and Mac specific instructions also.

Reply:
Yes, I've updated all instructions to start with Upper Case.